shaper/libs/parser.py

```diff
@@ -260,6 +267,12 @@ def write(self, data, path):
 
 class PropertyParser(TextParser):
 
+    @staticmethod
+    def __process_multiline_string(string):
+        if len(string.splitlines()) > 1:
```

Comment:
it's better to init additional variable for `string.splitlines()`

Reply:
done